xwiki-platform-core/xwiki-platform-legacy/xwiki-platform-legacy-events-hibernate/xwiki-platform-legacy-events-hibernate-ui/src/main/resources/XWiki/Notifications/Code/LegacyNotificationAdministrationTranslations.de.xml

```diff
@@ -36,10 +36,10 @@
   <minorEdit>false</minorEdit>
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
-  <content>legacy.event.migration.description=En la versi\u00F3n 12.6, XWiki pasa a un nuevo almacenamiento de eventos (notificaciones). Puesto que copiar eventos puede ser un proceso muy largo en wikis antiguas con registro de eventos y no siempre se desea mantenerlos, la elecci\u00F3n de hacerlo se deja al administrador de la wiki. No copiarlos implica que cualquier notificaci\u00F3n anterior parecer\u00E1 haber desaparecido. La migraci\u00F3n se ejecuta en segundo plano y no tienes que esperar a que termine para avanzar al siguiente paso.
+  <content>legacy.event.migration.description=XWiki hat seit der Version 12.6 einen neuen Speicherort f\u00FCr Ereignisse (Benachrichtigungen). Da das Transferieren der Ereignisdaten von dem alten in den neuen Speicher lange dauern kann, insbesondere f\u00FCr Wikis mit einer langen Nutzungszeit und vielen Ereignissen, und das Beibehalten alter Ereignisdaten nicht immer gew\u00FCnscht ist, wird diese Entscheidung Ihnen als Wiki Administrator \u00FCberlassen. Wenn Sie die Ereignisse nicht transferieren, wird es f\u00FCr Ihre Nutzer aussehen, als ob diese Ereignisse verschwunden sind. Der Transfer der Ereignisdaten findet in einem Hintergrundprozess statt und Sie brauchen die Beendigung nicht abzuwarten, sondern k\u00F6nnen gleich zum n\u00E4chsten Schritt gehen.
 ### Missing: legacy.event.count.store=Number of events in the standard store
 ### Missing: legacy.event.count.legacy=Number of events in the legacy store
-legacy.event.migration.alltime=Todos
-legacy.event.migration.since=Desde
-legacy.event.migration.start=Comenzar la migraci\u00F3n</content>
+legacy.event.migration.alltime=Der gesamte Zeitraum
+legacy.event.migration.since=Seit
+legacy.event.migration.start=Transfer starten</content>
```

Comment:
Those changes are weird in that PR

Reply:
Those are actually master changes, but compared to an older version of master. A merge of master should clean that.

Reply:
Ah indeed it's a conflicting file actually.